### PR TITLE
Good Night Disky

### PR DESCRIPTION
**Pull request by Xantholne**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/55940

## Why It's Good For The Game

Little disky is tired and needs bed too

You can tuck various things into bed for that immersion.

## Changelog
:cl:
add: You can now tuck disky into bed
add: You can now make beds by applying a bed sheet to them
add: You can now tuck in pai cards into bed
add: Added bed tucking element, can be added to any held object to allow tucking into beds
/:cl:

sleeptightdisker

### CODE REVIEW

**Comment by YakumoChen** [comment]
The conveyor meme is fine for turbo deadpop rounds since nobody wants to ERP with the disk on them, but I dont feel like we should outright encourage it outside of that circumstance. Theres criticism on the other (still open) PR that should be considered.

**Comment by YakumoChen** [comment]
image

**Comment by DeltaFire15** [comment]
Yeah, please do the same thing tg did and remove the way it influences loneop weight

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm [hunk 1]
```diff
@@ -634,7 +636,19 @@ This is here to make the tiles around the station mininuke change when it's arme
...
+	// How comfy is disky?
+		var/disk_comfort_level = 0
+
+		// Checking for items that make disky comfy
+		for(var/obj/comfort_item in loc)
+			if(istype(comfort_item, /obj/item/bedsheet) || istype(comfort_item, /obj/structure/bed))
+				disk_comfort_level++
+
+		if(disk_comfort_level >= 2) //Sleep tight, disky.
+			visible_message("<span class='notice'>[src] sleeps soundly. Sleep tight, disky.</span>")
```
Comment (silicons) [inline code/modules/antagonists/nukeop/equipment/nuclearbomb.dm:651]: this doesn't prevent the loneop stuff from triggering, right?

Comment (DeltaFire15) [reply code/modules/antagonists/nukeop/equipment/nuclearbomb.dm:651]: Huh, looks like it. Guess I was just hallucinating it having the return in there.
(They changed  it in the most recent commit, so yeah no more blocking loneops)

**Review by Linzolle — APPROVED** [review]

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm (hunk 1 above)
Comment (Linzolle) [reply code/modules/antagonists/nukeop/equipment/nuclearbomb.dm:651]: actually won't this end up triggering every single process() if it's stationary on a bed..?

**Pull request merged**